### PR TITLE
TMA-691 changed output to human readable part of colect data product action

### PR DESCRIPTION
**Pull request by Hahihula**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/681/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 12s
- gooddata-ruby-pronto-code-review-zuul : SKIPPED
- gooddata-ruby-unit-tests-zuul : SKIPPED

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/682/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 9s
- gooddata-ruby-pronto-code-review-zuul : SKIPPED
- gooddata-ruby-unit-tests-zuul : SKIPPED

File: deprecations.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Using `should_not` from rspec-expectations' old `:should` syntax without explicitly enabling the syntax is deprecated. Use the new `:expect` syntax or explicitly enable `:should` with `config.expect_with(:rspec) { |c| c.syntax = :should }` instead. Called from /home/hahihula/work/gooddata-ruby/spec/unit/actions/collect_data_product_spec.rb:9:in `block (2 levels) in <top (required)>'.
```
Comment (panjan) [inline deprecations.txt]: please remove this unrelated file

**Comment by kubamahnert** [comment]
Please include the jira task identifier in the merge request title as customary, thanks!

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/683/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/507/ : SUCCESS in 1m 55s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/793/ : SUCCESS in 2m 06s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/265/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/528/ : SUCCESS in 1m 57s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/805/ : SUCCESS in 1m 58s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1183/ : SUCCESS in 51m 33s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/941/ : SUCCESS in 37m 10s
